tests/integration/test_securitydata.py

```diff
@@ -39,13 +39,13 @@ class TestSecurityData:
     def test_get_all_plan_security_events(self, connection, plan_info):
         response_gen = connection.securitydata.get_all_plan_security_events(plan_info)
         for response in response_gen:
```

Comment:
Is `response_gen` not actually a generator?

Reply:
If this works like the other generators, I would expect `response` to be of type `Py42Response`

Reply:
At the end of `clients/securitydata.py::_get_security_detection_events()` there's this block:
```                if response.text:
                    cursor = response.data.get(u"cursor")
                    # if there are no results, we don't get a cursor and have reached the end
                    if cursor:
                        yield response, cursor
```
So the Py42Response is the first element in that tuple and the cursor is the second. My guess is that we updated the function without modifying the test but I haven't delved into the history to confirm that.

Reply:
Good catch!  we should also update the documentation of that method